src/main/java/com/iridium/iridiumskyblock/generators/VoidGenerator.java

```diff
@@ -15,7 +18,13 @@ public class VoidGenerator extends ChunkGenerator {
 
     @Override
     public @NotNull ChunkData generateChunkData(@NotNull World world, @NotNull Random random, int chunkX, int chunkZ, @NotNull BiomeGrid biomeGrid) {
-        return createChunkData(world);
+        final ChunkData chunkData = createChunkData(world);
+        for (int x = 0; x < 16; x++) {
+            for (int z = 0; z < 16; z++) {
+                biomeGrid.setBiome(x, z, Objects.requireNonNull(getSkyblockGenerator(world.getEnvironment()).biome.getBiome()));
+            }
+        }
```

Comment:
Possibly redundent

Reply:
i dont remember why this was here, but i remember it being important, so if it breaks you can just revert the change ig